### DISCUSSION TITLE
Virustotal free license: contribute new files

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13517] [General]
Hello
I want to use virustotal in a Security Onion installation in a business.
I'm wondering how to comply with this part of the virustotal license: "The Public API must not be used in business workflows that do not contribute new files."
Does anyone know how to comply with this? 
Thank you!

**Comment by dougburks** [comment]
If you have questions about VirusTotal licensing, then it might be best to ask VirusTotal themselves.

**Comment by Carlos-mb** [comment]
Yep
I agree... sorry. 
I tough that SO could be sending this files when using the api or something similar.

**Reply by dougburks** [reply]
Security Onion does not send files to VirusTotal by default.
